municipalities/beek-design-tokens/src/component/tilburg/search-card.tokens.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I expect the cards to look a bit more like the cards on https://www.gemeentebeek.nl/bestuur